demos/CMakeLists.txt

```diff
@@ -103,33 +90,17 @@ endfunction()
 
 include(CMakeParseArguments)
 
-#
-# ie_add_sample(NAME <target name>
-#               SOURCES <source files>
-#               [HEADERS <header files>]
-#               [INCLUDE_DIRECTORIES <include dir>]
-#               [DEPENDENCIES <dependencies>]
-#               [OPENCV_DENENDENCIES <opencv modules>]
-#               [EXCLUDE_CPPLINT]
-#
-macro(ie_add_sample)
-    set(options EXCLUDE_CPPLINT)
+# add_demo(NAME <target name>
+#     SOURCES <source files>
+#     [HEADERS <header files>]
+#     [INCLUDE_DIRECTORIES <include dir>]
+#     [DEPENDENCIES <dependencies>])
+macro(add_demo)
```

Comment:
omz_add_demo?

Reply:
Why? I don't see a reason to stay aligned with style of the old name. Plus this will conflict with `IE_SAMPLE*` naming which I didn't touch here.